pkg/controllers/sync/controller.go

```diff
@@ -338,24 +341,18 @@ func (s *SyncController) reconcile(qualifiedName common.QualifiedName) worker.Re
 
 	fedResource, possibleOrphan, err := s.fedAccessor.FederatedResource(qualifiedName)
 	if err != nil {
-		runtime.HandleError(errors.Wrapf(err, "Error creating FederatedResource helper for %s %q", kind, qualifiedName))
+		s.logger.WithValues("federated-type-kind", kind, "object", qualifiedName.String()).Error(err, "Failed to create FederatedResource helper")
```

Comment:
As usual, `"federated-type-kind"` is redundant.

Reply:
done, pls take review.